index.js

```diff
@@ -0,0 +1,185 @@
+// INDEX
+// - Variables
+// - Initialize
+// - Navigation
+// - Step 1
+// - Step 2
+// - Step 3
+
+
+
+const step0 = document.getElementById('step0');
+const step1 = document.getElementById('step1');
+const step2 = document.getElementById('step2');
+const step3 = document.getElementById('step3');
+const tax = 0.08;
+
+let customerBankAccount = 0;
+let customerSpendLimit = 0;
+let phoneChoice = "";
+let accessoryChoice = "";
+let bundleSize = "";
+let totalSpent = 0;
+let costPerBundle = 0;
+
+
+
+initialize();
+
+// *** each step gets its own display component for input and/or info ***
+
+function initialize() {
+  toggleDisplaySteps(step0, step1); //TODO - optional argument not working? had to add 2nd arg
+
+  document.getElementById('done-step0').onclick = () => { showStepOne(); }
+  document.getElementById('done-step1').onclick = () => { showStepTwo(); }
+  document.getElementById('done-step2').onclick = () => { showStepThree(); }
+
+  document.getElementById('back-step2').onclick = () => { toggleDisplaySteps(step1, step2); }
+  document.getElementById('back-step3').onclick = () => { toggleDisplaySteps(step2, step3); }
+}
+
+function showStepOne() {
+  toggleDisplaySteps(step1, step0);
+}
+
+function showStepTwo() {
+  getCustomerInitialInputs();
+
+  if(customerBankAccount === "" || customerSpendLimit == "") {
+     alert('Please fill out both inputs');
+     return;
+  } else {
+    addCustomerInputsToTopBar();
+    toggleDisplaySteps(step2, step1);
+  }
+}
+
+function showStepThree() {
+  collectPhoneData();
+  collectAccessoryData();
+  collectBundleData();
+
+  if(phoneChoice[0] === "" || accessoryChoice[0] === "" || bundleSize === "") {
+    alert('Please select one of each');
+    return;
+  } else {
+    combineDataForSummary();
+    moneyTotalsForSummary();
+  }
+  toggleDisplaySteps(step3, step2);
+}
+
+
+
+///
+//// NAVIGATION
+
+function toggleDisplaySteps(show, hide) {
+  show.classList.add("active");
+  hide.classList.remove("active");
```

Comment:
👍

Reply:
Thanks for the suggestion to get it away from a CSS update to a class add/remove function